src/main.rs

```diff
@@ -66,11 +66,11 @@ fn parse_matches() -> clap::ArgMatches {
                 .required(true),
         )
         .arg(
-            Arg::with_name("build_directory")
-                .long("build-dir")
+            Arg::with_name("cluster_data_path")
+                .long("cluster-data-path")
                 .takes_value(true)
-                .conflicts_with("local_path")
-                .help("Absolute path to build directory for release-channel
+                .required(true)
+                .help("Absolute path to cluster_data directory for storing accounts, genesis, etc
```

Comment:
Typically it's an antipattern for required args to have a flag, since they should just be positional arguments. On the flipside, these are super long commands, so I think you've done the right thing in this case

Reply:
ohhh didn't know that. good to know